runtime/common/src/parachains.rs

```diff
@@ -818,6 +970,117 @@ pub fn ensure_parachain<OuterOrigin>(o: OuterOrigin) -> result::Result<ParaId, B
 	}
 }
 
+
+/// Ensure that double vote reports are only processed if valid.
+#[derive(Encode, Decode, Clone, Eq, PartialEq)]
+pub struct ValidateDoubleVoteReports<T: Trait + Send + Sync>(rstd::marker::PhantomData<T>) where
```

Comment:
```suggestion
pub struct ValidateDoubleVoteReports<T>(rstd::marker::PhantomData<T>);
```

Trait bounds, if not required, should not be added to the definition.